data-products-dp0-3/table-access-and-queries.rst

```diff
@@ -117,6 +118,38 @@ The N:1 nature of these joins means that the data retrieved can contain columns
 be larger than expected, and take a long time to execute.
 
 
+.. _DP0-3-Table-Access-ADQL-passing-list:
+
+Advice for passing an existing list to a new query
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+LSST Query Services (Qserv) do not support subqueries. Thus, using subqueries is **not recommended** 
+although DP0.3 is not hosted on Qserv. Instead, when having a list of objects in hand either from a 
+previous query or a user-provided catalog, the list, formatted as a python tuple, can be passed to a 
+new query for table joins. The example query below is to retrieve information about individual observations 
+from the ``DiaSource`` and ``SSSource`` tables for indivdual unique objects selected from the ``SSObject`` 
+table and stored in a ``df_uniqueObj`` table from a previous query.
+
```

Comment:
Let's add a statement about how the example uses only three objects but lists can be very long (tens of thousands? did you test how long a list could be?)